systemd-units/bluechi-dep@.service

```diff
@@ -10,5 +10,6 @@ After=%i.service
 
 [Service]
 ExecStart=true
+ExecStop=true
```

Comment:
I don't think this actually fully solves the race condition. systemd will first run ExecStop commands, but unless the process died it will still send kill signals after that.

Reply:
Maybe `KillMode=none` instead?

Reply:
Yes, it didn't fully solve the race condition. 
I'd use your proposed solution to just use `ExecStop=/bin/true`. I think additionally having `SuccessExitStatus=` doesn't hurt, but also shouldn't have any effect - so just sticking with `ExecStop=` should be best. What do you think? @alexlarsson